src/patterns/components/lists/angular/code/indented-list.hbs

```diff
@@ -2,15 +2,12 @@
   Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed gravida urna quis nulla ultrices, sed efficitur risus elementum.
 </p>
 
-<sprk-unordered-list listType="indented">
-  <sprk-list-item>
-    Indented Item
+<sprk-unordered-list listType="indented" idString="unordered-list-3">
+  <sprk-list-item idString="list-item-3">
+    fish
```

Comment:
These need to match the documentation site. So, that when users copy the code blocks they get what they are seeing in the docs. This needs to stay as "Indented Item".